en/docs/devportal/api-workflows/consuming-api-workflows.md

```
@@ -0,0 +1,100 @@
+
+---
+
+## How AI Agents Discover Workflows
+
+Agents that interact with the developer portal can discover workflows through two machine readable endpoints:
+
+### `llms.txt`
+
+The portal's `llms.txt` file provides a structured index of everything the portal exposes to AI agents, including a list of all published workflows with agent visibility set to **Visible**. Each workflow entry includes its name, description, and a link to its agent prompt and Arazzo specification.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)